### PR TITLE
Add the needed config to promote the GRC image

### PR DESCRIPTION
**Pull request by vboulos**

### CODE REVIEW

File: ci-operator/config/stolostron/acmqe-grc-test/stolostron-acmqe-grc-test-release-2.7.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+images:
+- dockerfile_path: Dockerfile.interop
```
Comment (ascerra) [inline ci-operator/config/stolostron/acmqe-grc-test/stolostron-acmqe-grc-test-release-2.7.yaml:7]: it looks like the Dockerfile in this repo is `Dockerfile` not `Dockerfile.interop`

https://github.com/stolostron/acmqe-grc-test/tree/release-2.7

Comment (vboulos) [reply ci-operator/config/stolostron/acmqe-grc-test/stolostron-acmqe-grc-test-release-2.7.yaml:7]: oh I see, I will ask the GRC team to merge the code and then we will be able to see the Dockerfile.interop

**Comment by ascerra** [comment]
/pj-rehearse pull-ci-stolostron-acmqe-grc-test-release-2.7-images

File: ci-operator/config/stolostron/acmqe-grc-test/stolostron-acmqe-grc-test-release-2.7.yaml (hunk 1 above)
Comment (ascerra) [reply ci-operator/config/stolostron/acmqe-grc-test/stolostron-acmqe-grc-test-release-2.7.yaml:7]: We could just use Dockerfile here instead, that's up to you though.

Comment (vboulos) [reply ci-operator/config/stolostron/acmqe-grc-test/stolostron-acmqe-grc-test-release-2.7.yaml:7]: better to use the interop one for OPP tests, not sure if the other one is tested

Comment (vboulos) [reply ci-operator/config/stolostron/acmqe-grc-test/stolostron-acmqe-grc-test-release-2.7.yaml:7]: /ascerra The PR is merged and the Dockerfile.interop is now in the release-2.7 branch

**Comment by ascerra** [comment]
/pj-rehearse pull-ci-stolostron-acmqe-grc-test-release-2.7-images

**Comment by ascerra** [comment]
The image build rehearsal job ran and passed this /lgtm

**Review by ascerra — APPROVED** [review]

**Review by Kyl-Bempah — APPROVED** [review]
LGTM

**Pull request merged**